src/core/server/ui_settings/create_or_upgrade_saved_config/create_or_upgrade_saved_config.test.ts

```diff
@@ -136,20 +135,21 @@ describe('uiSettings/createOrUpgradeSavedConfig', function() {
       });
 
       await run();
-      sinon.assert.calledOnce(logWithMetadata);
-      sinon.assert.calledWithExactly(
-        logWithMetadata,
-        ['plugin', 'elasticsearch'],
-        sinon.match('Upgrade'),
-        sinon.match({
-          prevVersion,
-          newVersion: version,
-        })
-      );
+      expect(loggingServiceMock.collect(logger).debug).toMatchInlineSnapshot(`
+        Array [
+          Array [
+            "Upgrade config from 4.0.0 to 4.0.1",
+            Object {
+              "newVersion": "4.0.1",
```

Comment:
don't even know why we need those as meta data. message string already contains both versions